virttest/utils_test/qemu/__init__.py

```diff
@@ -657,27 +657,23 @@ def hotplug_memory(self, vm, name):
     def unplug_memory(self, vm, name):
         """
         Unplug memory device
-        step 1, unplug dimm device
+        step 1, unplug the associated dimm or virtio-mem device
         step 2, unplug memory object
 
         :param vm: VM object
         :param name: memory device name
         """
         devices = []
-        qid_dimm = "dimm-%s" % name
+        qid_list = ["dimm-%s" % name, "virtio_mem-%s" % name]
```

Comment:
I have a question: will dimm and virtio-mem use the same `name` in the case? And can they be used together? If no, means here will always log a warning message, right?

Reply:
Yes, there will always be a warning in the logs. Even if you try to use the same name twice in the `target_mems`, for virtio-mem is needed to specify the `vm_memdev_model`, that will lead to a duplicate ID.

Reply:
No warning message anymore after applying Xu's suggestion